### PR TITLE
use wildcard in tag references

### LINKED ISSUE #369: use wildcard support in tags
replace combiners with new wildcards where appropriate

example:
https://github.com/BAMWelDX/weldx/blob/eb6ef561128aa71a00d0376cf7e72584f329e92c/weldx/asdf/schemas/weldx.bam.de/weldx/datamodels/single_pass_weld-1.0.0.schema.yaml#L143-L155

### PR DESCRIPTION
**Pull request by CagtayFabry**
## Changes
- replace references to base asdf tags with `-1.*` version wildcard
- update `single-pass-weldx.1.0.0.schema` to allow groove types by wildcard

## Related Issues

Closes #369 

## Checks

- [x] updated CHANGELOG.md
- [ ] updated tests
- [x] updated doc/
- [x] update example/tutorial notebooks

### CODE REVIEW

File: weldx/asdf/schemas/weldx.bam.de/weldx/datamodels/single_pass_weld-1.0.0.schema.yaml [hunk 1]
```diff
@@ -140,23 +140,11 @@ properties:
...
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/DVGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/FFGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/HUGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/HVGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/IGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/UGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/UVGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/VGroove-1.0.0"
-              - tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/VVGroove-1.0.0"
+            tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/*-1.0.0"
```
Comment (vhirtham) [inline weldx/asdf/schemas/weldx.bam.de/weldx/datamodels/single_pass_weld-1.0.0.schema.yaml:143]:
```suggestion
            tag: "tag:weldx.bam.de:weldx/groove/iso_9692_1_2013_12/*-1.*"
```

**Review by vhirtham — APPROVED** [review]

**Comment by CagtayFabry** [comment]
@vhirtham please update branch and merge when appropriate with the other PRs :)

**Comment by vhirtham** [comment]
@CagtayFabry Got to fix two Spinx warnings that caused the documentation build to fail. However, I couldn't reproduce it on my machine. Can you check if you can reproduce it on your computer by using a commit before my fixes? The warnings aren't even related to the changes in this branch and originate from the scipy docs via inheritance.

**Comment by CagtayFabry** [comment]
> @CagtayFabry Got to fix two Spinx warnings that caused the documentation build to fail. However, I couldn't reproduce it on my machine. Can you check if you can reproduce it on your computer by using a commit before my fixes? The warnings aren't even related to the changes in this branch and originate from the scipy docs via inheritance.

probably unrelated to our code but to the new [scipy 1.7 release](https://github.com/scipy/scipy/releases/tag/v1.7.0)

**Pull request merged**